<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Seleziona_Prodotto_Window</class>
 <widget class="QWidget" name="Seleziona_Prodotto_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Prodotto</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QComboBox" name="prodotti_comboBox">
   <property name="geometry">
    <rect>
     <x>87</x>
     <y>40</y>
     <width>376</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox::down-arrow {

	image: url(:/arrows/res/down_arrow.png);
        width: 14px;
        height: 14px;

        }
        QComboBox::drop-down
        {
        border: 0px; /* This seems to replace the whole arrow of the combo box */
        color:black

        }

QComboBox {
    color: black;
border-radius:30px;
border: 5px solid rgb(0, 134, 255);
	selection-color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtTop</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>320</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);
image: url(:/sfondo_microone/res/pagina_MICROONE.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>195</x>
     <y>0</y>
     <width>210</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 134, 255);

color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Seleziona Prodotto</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>17</x>
     <y>40</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/hand/res/Hand.png);
      border: 5px solid rgb(0, 134, 255);
border-radius:30px;
     </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn_1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>0</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);
                    color: rgb(255, 255, 255);
                    border-radius:10px;
                </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img_resources.qrc">
     <normaloff>:/cancel/res/cancel.png</normaloff>:/cancel/res/cancel.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn_">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 140, 255);
                 color: rgb(255, 255, 255);
                 border-radius:10px;
             </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="img_resources.qrc">
     <normaloff>:/cancel/res/cancel.png</normaloff>:/cancel/res/cancel.png</iconset>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="avanti_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>120</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 134, 255);

color: rgb(255, 255, 255);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>Avanti</string>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>prodotti_comboBox</zorder>
  <zorder>label_2</zorder>
  <zorder>cancel_btn_1</zorder>
  <zorder>cancel_btn_</zorder>
  <zorder>avanti_btn</zorder>
  <zorder>label</zorder>
 </widget>
 <resources>
  <include location="img_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancel_btn_</sender>
   <signal>clicked()</signal>
   <receiver>Seleziona_Prodotto_Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
